x-pack/legacy/plugins/ml/public/application/services/ml_api_service/datavisualizer.js

```diff
@@ -33,7 +33,7 @@ export const fileDatavisualizer = {
     const { index, data, settings, mappings, ingestPipeline } = obj;
 
     return http({
-      url: `${basePath}/file_data_visualizer/import${paramString}`,
+      url: `${basePath}/fileupload/import${paramString}`,
```

Comment:
Switching to the `fileupload` route for the import means we are losing the telemetry on the count of indexes created by the ML file data visualizer, which was done by a call to `incrementFileDataVisualizerIndexCreationCount` in `/ml/server/lib/ml_telemetry/ml_telemetry.ts`. We need to keep the telemetry part of the import, but get the file upload plugin to do the actual file import.